README.md

```diff
@@ -147,12 +147,22 @@ log in Xcode's report navigator and expanding the "Compile documentation" step.
   
 ## Using `docc` to build and preview documentation
 
-You can use `docc` directly to build documentation for your Swift framework
-or package. The below instructions use this repository as an example but
-apply to any Swift package. Just replace any reference to `SwiftDocC` below
-with the name of your package.
+### Using the Swift-DocC plugin (preferred)
 
-### 1. Generate a symbol graph file
+The preferred way of building documentation for your Swift package is by using
+the Swift-DocC Plugin. Refer to instructions in the plugin's 
+[documentation](https://apple.github.io/swift-docc-plugin/documentation/swiftdoccplugin/)
+to get started with [building](https://apple.github.io/swift-docc-plugin/documentation/swiftdoccplugin/generating-documentation-for-a-specific-target), [previewing](https://apple.github.io/swift-docc-plugin/documentation/swiftdoccplugin/previewing-documentation),
+and publishing your documentation to your [website](https://apple.github.io/swift-docc-plugin/documentation/swiftdoccplugin/generating-documentation-for-hosting-online) or [GitHub Pages](https://apple.github.io/swift-docc-plugin/documentation/swiftdoccplugin/publishing-to-github-pages).
+
+### Assembling symbol graphs and building with `docc` directly
```

Comment:
I'm not sure any of this belongs in the README at this point. Maybe we should move it to the CONTRIBUTING if it's not already there?

Reply:
that's a good point. it's more of an advanced use case. I've moved it